src/component.js

```diff
@@ -75,7 +90,12 @@ export function insertPptm(env : string = getEnv(), isDebug : boolean = getDebug
     // website, and then it'll render an iframe from the PayPal domain which will in turn
     // initialize the SDK again. We don't want to insert another pptm.js on the paypal.com
     // domain, though.
-    if (!_isPayPalDomain()) {
+
+    // We do not want to load pptm.js for shopping component. Merchant may always add muse, checkout button  to load pptms.js if neeeded.
+    // However, shopping component will be a replacement of pptm.js. Some merchant may use it to publish store cash events.
+    // https://engineering.paypalcorp.com/jira/browse/DTSHOPSDK-295
+    
+    if ((!_isPayPalDomain()) && (!_isShoppingSdkComponent())) {
```

Comment:
Should this not be an `||`? As an `&&` the original conditional `(!_isPayPalDomain())` will no longer act as it did before.

Reply:
it is negation that makes it a bit confusing. What we trying to do here is: 
either `_isPayPalDomain()` || `_isShoppingSdkComponent()` => do nothing
or 
`!_isPayPalDomain()` && `!_isShoppingSdkComponent()` => load the muse
so the logic is fine here